client/source/components/presentational/Route.jsx

```diff
@@ -1,64 +1,57 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-class Route extends React.Component {
-  constructor({ props, route }) {
-    super(props);
+const Route = ({ route, handleEdit }) => {
+  const divStyle = {
```

Comment:
This is fine for now, but I think in the long run, it will be better to refactor this so the element has a class that can be styled in the CSS stylesheet.